src/components/Button/Button.tsx

```diff
@@ -36,6 +44,14 @@ export const Button = (
     className,
   } = props
 
+  let isBig = big || false
+  let isSmall = small || false
+  if (size === 'big') {
+    isBig = true
+  }
+  if (size === 'small') {
+    isSmall = true
+  }
```

Comment:
I think another way to do this is  
``` 
const isBig = size === 'big' || (!size && big)
const isSmall = size === 'small' || (!size && small)
```

Reply:
That's assuming we want to ignore the deprecated props if they pass down `size`. @suzubara I'm blanking on what we decided here... I know we said we wouldnt fix if they passed down `big small` together but do we want to guard against `size='big' small` ?

Reply:
Also, I very much like your suggestion, but I have a counter suggestion.  This is slightly into code golf territory, but how would you feel about

```typescript
  const isBig = big ? big : size === 'big'
  const isSmall = small ? small : size === 'small'
```

It changes things to prefer the old `big` and `small` in case of both, but that feels appropriate to me

Reply:
Or to prefer `size`
```typescript
  const isBig = size ? size === 'big' : big
  const isSmall = size ? size === 'small' : small
```

Reply:
I like the last option (that prefers the `size` prop). I don't think we need to deal with guarding against both `big small`, that is their choice and will be deprecated 🤷‍♀️